cockatrice/src/dlg_settings.cpp

```diff
@@ -269,8 +282,9 @@ void GeneralSettingsPage::retranslateUi()
     defaultUrlLabel.setText(tr("Primary download URL:"));
     fallbackUrlLabel.setText(tr("Fallback download URL:"));
     urlLinkLabel.setText(QString("<a href='%1'>%2</a>").arg(WIKI_CUSTOM_PIC_URL).arg(tr("How to set a custom picture url")));
-    clearDownloadedPicsButton.setText(tr("Reset/Clear Downloaded Pictures"));
-    updateNotificationCheckBox.setText(tr("Notify when new client features are available"));
+    clearDownloadedPicsButton.setText(tr("Reset/clear downloaded pictures"));
+    updateReleaseChannelLabel.setText(tr("Update channel"));
+    updateNotificationCheckBox.setText(tr("Notify when a new version is available"));
```

Comment:
I've tried to change the sentence, but it's so long that it forces the settings window to be wider. So the candidates are:
```
# original wording
Notify when new client features are available
# short, misleading wording
Notify when a new version is available
# Precise but too long wording
On connect, notify everytime a feature supported by the server is missing in my client
# Other attempts
Notify if a feature supported by the server is missing in my client
Notify if the server reports that a feature is missing in my client
```
Others?